source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -36,90 +36,98 @@ By default, MinIO requires authentication to scrape the metrics endpoints.
 To generate the needed bearer tokens, use :mc:`mc admin prometheus generate`.
 You can also disable metrics endpoint authentication by setting :envvar:`MINIO_PROMETHEUS_AUTH_TYPE` to ``public``.
 
-MinIO provides the following scraping endpoints, relative to the base URL:
+You can also access metrics using :mc-cmd:`mc admin prometheus metrics` and the metric type for the desired category.
+For more information, see the :mc-cmd:`MinIO Admin Client reference <mc admin prometheus metrics>`.
```

Comment:
There should be a mention of getting metrics with `mc`, perhaps there's a better location for this text however.

Reply:
I like that. 👍🏻